R/cloud.R

```diff
@@ -199,7 +201,10 @@ cloud_check <- function(pkg = ".", tarball = NULL, revdep_packages = NULL, r_ver
 
   cli_alert("Run {.fun cloud_status} to monitor job status")
 
-  cloud_job(job_id)
+  cloud_job(job_id = job_id)
+  cloud <- dir_find(pkg, "cloud")
+  out_dir <- file.path(cloud, job_id)
+  dir.create(out_dir, showWarnings = FALSE, recursive = TRUE)
```

Comment:
How should this play out for the case where a `tarball` *was* passed in?

Reply:
I don't understand why you changed this code at all?

Reply:
I don't think the tarball should change anything with this? passing the tarball just avoids building the package before sending it.

Reply:
OK. I've never used it this way (with a tarball). I was wondering if people passing in a tarball might not be in a reasonable directory for storing the results locally. But even if this isn't quite right, we can refine it later.